### ISSUE TITLE
Format misorders variadic/exported operators

### ISSUE THREAD

**Issue by nk9** [casey/just#1449]
This is valid just:

```
recipe +$f:
    echo "v"
```

But if you run this through the `--fmt` formatter, it changes this:

```
$ just --unstable --fmt --justfile test.just
Wrote justfile to `/tmp/test.just`
$ cat test.just
recipe $+f:
    echo "v"
```

And when you try to run that, `just` complains:

```
$ just --list --justfile test.just
error: Expected identifier, but found '+'
  |
1 | recipe $+f:
  |         ^
```

**Comment by casey** [comment]
Nice catch! Fixed in #1451.

**Comment by nk9** [comment]
Talk about a quick fix! Thanks. 😄
